### ISSUE TITLE
Organizer can selectively add or remove attendees

### ISSUE THREAD

**Issue by sidhene** [railsbridge/bridge_troll#456]
I think Meetup had this feature but Bridgetroll does not. 

There are various reasons why an organizer might want to mange students or volunteers, such as promoting a student who has been stuck on the waitlist too many times or to ensuring gender balance when more men outnumber women volunteers. This feature would allow them to do that.

**Comment by sidhene** [comment]
I'm poking around in the code, and it looks like the easiest and safest way is to edit the waitlist position. Then the organizer can either expand the RSVP limit or wait for successful RSVPs to cancel.

**Comment by sidhene** [comment]
Then canceling an rsvp (which socially could prove problematic) can be another story.

**Comment by lilliealbert** [comment]
RSVPs can be removed from the "Show All Attendee RSVP Details" panel on the organizer console, so that part is already done.

**Comment by lilliealbert** [comment]
If we were to do the "Selectively Add Attendees" feature, I would rename it to "Selectively Promote Attendees From The Waitlist" because I don't think it makes sense to wholesales create an RSVP for someone — they should have shown some indication that they'd like to attend.

**Comment by sidhene** [comment]
Thanks @lilliealbert! This just allows the organizer to destroy RSVPs though, right? It doesn't let you move someone from waitlist to attendee (which would probably be blocked by the attendance limit) or let you edit the waitlist rank.

Correct me if I'm wrong.

**Comment by lilliealbert** [comment]
Yes, maybe you didn't see my second comment, above? Or was that not clear?

> If we were to do the "Selectively Add Attendees" feature, I would rename it to "Selectively Promote Attendees From The Waitlist" because I don't think it makes sense to wholesales create an RSVP for someone — they should have shown some indication that they'd like to attend.

**Comment by sidhene** [comment]
I hadn't seen the second comment yet. Yes, selectively promoting attendees from the waitlist would do the job. I think this is needed and within my ability so I'd like to start development soon.

What do we want to do about the potential conflict of the RSVP limit?

**Comment by kmolo** [comment]
A (very) small use case for a "Selectively Add Attendees" features may be for this proposed feature:

"Restrict RSVP to those who have previously attended another Bridgetroll event"
https://github.com/railsbridge/bridge_troll/issues/438

...this solves the problem of what if someone has the appropriate background to directly attend an advanced event, but didn't gain that experience thru a BridgeTroll-listed event.

I could imagine something similar to the "Manage Organizers" functionality that allows you to directly add Organizers who haven't RSVP'ed as Volunteers.  

Of course, with the appropriate text to encourage folks not to use this functionality too liberally -- I agree with @lilliealbert that if folks can't take the time to BridgeTroll to sign up, they may have a higher no-show rate.
